recipe/meta.yaml

```diff
@@ -1,37 +1,31 @@
 {% set name = "streamlit-extras" %}
-{% set version = "0.2.7" %}
+{% set version = "0.4.0" %}
```

Comment:
Just a note, unless there's a specific need for 0.4.0, then 0.4.2 should be built now

Reply:
I believe that snowflake asked for specifically `0.4.0`.

Reply:
because they requested `0.4.0` on April 15th, well after `0.4.2` was released, I would keep `0.4.0`.